<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>811</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUrl">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>http://192.168.1.222:8088/wlis3/j_acegi_security_check.action?ajax=true</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入URL地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVerificationCode">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRefresh">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPwd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>111111</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnPost">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送(Post)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnClear">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清 空</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVerficationCode">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUserName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnGet">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>获取(Get)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>31</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>验证码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditData">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>521</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送数据：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
